CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ weight: 10
 ---
 </pre>
 
-- The title will be shown on the page, but the description will only be used in the description meta tag, so say "Zen Cart" somewhere in the description.  
+- The title will be shown on the page, but the description will only be used in the description meta tag and the subtitle for the page when on the folder is being browsed. 
```

Comment:
`when on the folder is being browsed.`

Feels like a grammar error here.

Reply:
- The title will be shown on the page, but the description will not; it is only shown as the subtitle for the page when the parent folder is being browsed.